source/02-EV/02.ptx

```diff
@@ -387,7 +387,7 @@ What can you conclude about
 <li>A set of 3 vectors that does not span <m>\mathbb{R}^3</m></li>
 <li>A set of 4 vectors that spans <m>\mathbb{R}^3</m></li>
 </ul>
-                    For any of the sets you constructed that did span the required space, are any of the vectors a linear combination of the others in your set? 
+                    For any of the sets you constructed that did span the required vector space, are any of the vectors a linear combination of the others in your set? 
```

Comment:
The rest of the changes are in this spirit -- removing usage of the word 'space' without further clarification, usually in favor of 'vector space'.